features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -149,6 +152,21 @@ internal class RumViewManagerScope(
                 iterator.remove()
             }
         }
+
+        if (event is RumRawEvent.AddViewLoadingTime && !hasActiveView) {
+            sdkCore.internalLogger.log(
+                InternalLogger.Level.WARN,
+                InternalLogger.Target.USER,
+                { NO_ACTIVE_VIEW_FOR_LOADING_TIME_WARNING_MESSAGE }
+            )
+            sdkCore.internalLogger.logApiUsage {
+                InternalTelemetryEvent.ApiUsage.AddViewLoadingTime(
+                    overwrite = event.overwrite,
+                    noView = hasNoView,
+                    noActiveView = !hasNoView
```

Comment:
This is a bit confusing to me. If we get here then it means we're inside a block where hasActiveView is necessarily false, but we could end up passing false to AddViewLoading time for noActiveView indicating that there is actually an active view?

Reply:
Here I'm replicating the intended usage of this telemetry, where only one of the two attributes can be true at a time.